files/BatteryTools.py

```diff
@@ -20,11 +20,20 @@ def battery_model_sizing(model, desired_power, desired_capacity, desired_voltage
         volts
     :param size_by_ac_not_dc: optional bool
         Sizes for power and capacity are on AC side not DC side of battery-inverter regardless of connection type
+    :param module_dict: optional dict
+        Module specs for scaling surface area
+            capacity: float
+                Utility-scale module capacity for scaling surface area
+                Battery: kWhAC if AC-connected, kWhDC otherwise
+                BatteryStateful: battery kWhDC
+            surface_area: float
+                Utility-scale module surface area m^2
+                m^2 of module battery
     """
```

Comment:
I would do an error check on module_specs and make sure both required keys exist, otherwise raise ValueError. This way the user doesn't think it's worked if it hasn't